src/kernel/task/resource/sig_table.c

```diff
@@ -32,6 +32,15 @@ static void task_sig_handler_ignore(int sig) {
 #define Core task_sig_handler_terminate /* FIXME */
 #define Stop task_sig_handler_terminate /* FIXME */
 #define Cont task_sig_handler_ignore    /* FIXME */
+
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Winitializer-overrides"
```

Comment:
This is very difficult can be make it easy?
If I right understand this problem, there are some syscall numbers with identical id for example 
```
#define SIGABRT     6       /* (ANSI)     Abort.  */
#define SIGIOT      6       /* (4.2 BSD)  IOT trap.  */
```
I think right way to fix this warning rewrite initialization array